codec-http/src/main/java/io/netty/handler/codec/http/HttpRequestEncoder.java

```diff
@@ -38,7 +39,7 @@ public boolean acceptOutboundMessage(Object msg) throws Exception {
     @Override
     protected void encodeInitialLine(ByteBuf buf, HttpRequest request) throws Exception {
         AsciiString method = request.method().name();
-        buf.writeBytes(method.array(), method.arrayOffset(), method.length());
```

Comment:
@trustin Agreed it introduced more complexity...but allows for us to wrap subregions as I discussed.  I think we are in agreement that I should submit a follow-up PR to put `arrayOffset()` back in?